data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,59 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Saryd"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "saryd" "folklore"
+	on offer
+		conversation
+			`Walking through the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`
+			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" he asks.`
+			choice
+				`	"What sort of celebration?"`
+				`	"Sorry, I'm busy right now."`
+					decline
+			`	"A celebration of Saryd folklore, it is," he explains. "Though for a long time among the stars we've lived, timeless classics from before our spacefaring age, we still celebrate. A set of theatrical plays about some of them, today we are hosting. Watch them for free, anyone can."`
+			choice
+				`	"I've always got time for a story. Lead the way."`
+				`	"I'm not really up for a story now. I'll pass."`
+					decline
+			`	You follow him outside the spaceport city and deep into a nearby forest. On the way there, several other Saryds in similar ancient garbs are guiding others who were also intrigued by the offer.`
+			`	You arrive at a clearing with three different paths to take, and you can make out a faint light from deep within each of them. After a few minutes, it seems as if all the Saryds involved in the celebration have arrived. They grab three caskets from behind a bush and lay them before you and the others they brought here.`
+			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
+			choice
+				`	(Grab a seed.)`
+					goto seed
+				`	(Grab a seashell.)`
+					goto pearl
+				`	(Grab a feather.)`
+			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and you do so.`
```

Comment:
"and you do so" feels unnecessary to me. I think leaving it off sounds better. I think it can be safely assumed that you take the path you were pointed to without extra detail saying that you do so.
```suggestion
			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right.`
```
This same change would be made to the other two choices.